### PR TITLE
fix bug that getStationServicesByID cannot be found

### PR DESCRIPTION
**Pull request by belteus**

### CODE REVIEW

File: src/routes/stations.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ router.get('/getByKeyword/:lat,:long,:keyword', function (req, res) {
  * @apiSuccess {String} Services.Type type of the services
  * @apiSuccess {String} Services.Title title of the services
  */
-router.get('/getStationServicesById/:id,', function (req, res) {
```
Comment (boybundit) [inline src/routes/stations.js:107]: , 😏

Comment (belteus) [reply src/routes/stations.js:107]: :flushed::flushed::flushed:

**Comment by boybundit** [comment]
@belteus We should have unit test for this! 😜

**Comment by boybundit** [comment]
@belteus Deployment is currently not possible as my Azure hits spending limit... The new monthly billing period will start on Nov 21.

**Pull request merged**